account-kit/react/src/components/auth/sections/InjectedProvidersAuth.tsx

```diff
@@ -4,7 +4,6 @@ import { useConnect } from "../../../hooks/useConnect.js";
 import { Button } from "../../button.js";
 import { useAuthContext } from "../context.js";
 
-// eslint-disable-next-line jsdoc/require-jsdoc
```

Comment:
Does this not apply anymore?

Reply:
now that all the exported methods are correctly being parsed, we shouldn't need to disable this lint rule anymore